third_party/move/move-model/src/lib.rs

```diff
@@ -317,54 +314,6 @@ fn collect_related_modules_recursive<'a>(
     }
 }
 
-/// Build a `GlobalEnv` from a collection of `CompiledModule`'s. The `modules` list must be
-/// topologically sorted by the dependency relation (i.e., a child node in the dependency graph
-/// should appear earlier in the vector than its parents).
-pub fn run_bytecode_model_builder<'a>(
```

Comment:
what was the motivation in removing this?

Reply:
> 

Same as above. Dead code.